core/frontend/src/test/tile/map/MapLayerFormat.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import {
+  ImageryMapLayerTreeReference,
+  MapLayerFormat,
+  MapLayerFormatRegistry,
+  MapLayerImageryProvider,
+} from "../../../tile/internal";
+import { ImageMapLayerProps, ImageMapLayerSettings } from "@itwin/core-common";
+import { IModelConnection } from "../../../IModelConnection";
+
+class TestMapLayerFormat extends MapLayerFormat {
+  public static override formatId = "TestMapLayerFormat";
+
+  public static override createImageryProvider(settings: TestMapLayerSettings): MapLayerImageryProvider | undefined {
+    return new TestMapLayerImageryProvider(settings, false);
+  }
+
+  public static override createMapLayerTree(layerSettings: TestMapLayerSettings, layerIndex: number, iModel: IModelConnection) {
+    return new ImageryMapLayerTreeReference({ layerSettings, layerIndex, iModel });
```

Comment:
The constructor you are calling is `internal`.
```
export class ImageryMapLayerTreeReference extends MapLayerTileTreeReference {
    constructor(layerSettings: MapLayerSettings, layerIndex: number, iModel: IModelConnection);
    // @internal
    constructor(args: {
```

Reply:
Sorry. Must have forgotten to actually remove the `internal` tag after debating about this...